### DISCUSSION TITLE
阿里云服务器部署doris，be能收到fe的心跳检测，但是BE连接FE失败（通过127.0.0.1:9020）。这里为什么是127.0.0.1？在内网同一网段下部署就没有这个问题

### DISCUSSION THREAD

**Discussion by GavinWangxj** [apache/doris#10037] [A - General / Q&A]
BE:10.27.186.5 日志
I0609 16:45:02.145465 17409 heartbeat_server.cpp:58] get heartbeat from FE.host:127.0.0.1, port:9020, cluster id:2099112140, counter:37
W0609 16:45:06.074265 17294 utils.cpp:90] fail to get master client from cache. host=127.0.0.1, port=9020, code=7

FE:10.46.127.99    backends信息
            BackendId: 14008
              Cluster: default_cluster
                   IP: 10.27.186.5
        HeartbeatPort: 9050
               BePort: 9060
             HttpPort: 8040
             BrpcPort: 8060
        LastStartTime: 2022-06-09 16:40:36
        LastHeartbeat: 2022-06-09 16:44:47
                Alive: true
 SystemDecommissioned: false
ClusterDecommissioned: false
            TabletNum: 0
     DataUsedCapacity: .000
        AvailCapacity: 1.000 B
        TotalCapacity: .000
              UsedPct: 0.00 %
       MaxDiskUsedPct: 0.00 %
               ErrMsg:
              Version: 0.14.7-Unknown
               Status: {"lastSuccessReportTabletsTime":"N/A"}

BE conf配置
priority_networks = 10.27.186.5/24

FE conf配置
priority_networks = 10.46.127.99/24

**Comment by FreeOnePlus** [comment]
Hello, your configuration item is misconfigured:
BE conf configuration
priority_networks = 10.27.186.5/24
FE conf configuration
priority_networks = 10.46.127.99/24

It should be configured as follows:
BE conf configuration
priority_networks = 10.27.186.0/24
FE conf configuration
priority_networks = 10.46.127.0/24

After the modification, please clear the FE metadata folder and the BE data folder, and then restart. If you have any questions, please feedback~
